.travis.yml

```diff
@@ -54,16 +65,23 @@ before_script:
   - psql -U postgres -c 'create database test'
   - psql -U postgres -d test -c 'create extension postgis'
   - gunzip < tests/data/hotosm_bdi_waterways.sql.gz | psql -U postgres test
+  # load cite data
+  - python3 tests/load_es_data.py tests/cite/ogcapi-features/canada-hydat-daily-mean-02HC003.geojson IDENTIFIER
 
 script:
   - pytest --cov=pygeoapi
   - make -C ./docs html
   - find . -type f -name "*.py" | xargs flake8
+  # run docker image with cite configuration
+  - docker run -d -p 5001:5001 --network host --add-host="localhost:127.0.0.1" --rm -it -v $(pwd)/tests/cite/ogcapi-features/cite.config.yml:/pygeoapi/local.config.yml --name pygeoapi-travis-master geopython/pygeoapi:latest run
+  - docker ps | grep -wq "pygeoapi-travis-master"
```

Comment:
OK, this fails now. Rethinking, this is a bit complex to setup as services on host are needed from within the Docker Container, always very complex and error-prone. Better via a Docker compose-file like the [demo one](https://github.com/geopython/demo.pygeoapi.io/blob/master/services/pygeoapi_cite/docker-compose.yml). Also the `docker logs | grep` is not a good test I realize. Possibly better to just 

1) start the Docker image, wait til ready
2) do sanity check with some `curl `requests
2) stop the Docker image

But the Unit tests all run ok with Docker. That already gives confidence. So I suggest to skip for now and open a new issue to implement robust Docker tests for Travis.  For now, if you remove the Cite Docker commands, all Travis should be ok, and we can merge!

Reply:
Well I understand that your plan is good but more inclined to an e2e strategy which can be opened as a different issue. Everything is now passing so I would merge.